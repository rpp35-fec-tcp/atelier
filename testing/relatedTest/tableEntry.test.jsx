/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import TableEntry from '../../client/src/components/relatedItems&comparison/tableEntry';

describe('TableEntry Component', () => {
  it('snapshot test', () => {
    const modal = renderer.create(<TableEntry />).toJSON();
    expect(modal).toMatchSnapshot();
  });
});
