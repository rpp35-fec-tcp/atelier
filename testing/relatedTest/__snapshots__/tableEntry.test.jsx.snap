// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableEntry Component snapshot test 1`] = `
<tr>
  <td />
  <td
    className="description"
  />
  <td />
</tr>
`;
